### PR TITLE
LPD-20908 Use BufferedReader instead of IOUtius

### PR DESCRIPTION
**Pull request by sillascavalcanti**
[Issue](https://liferay.atlassian.net/browse/LPD-20908)

### CODE REVIEW

**Comment by sillascavalcanti** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
Hey @sillascavalcanti, how are you?

Just to remind you that when you open a PR you need to run `ci:test:analytics-cloud-acceptance`, okay?

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

File: modules/dxp/apps/osb/osb-faro/osb-faro-engine-client/src/main/java/com/liferay/osb/faro/engine/client/web/client/ResponseErrorHandler.java [hunk 1]
```diff
@@ -37,7 +40,9 @@ public void handleError(ClientHttpResponse clientHttpResponse)
 		}
 
 		try (InputStream inputStream = clientHttpResponse.getBody()) {
-			String response = IOUtils.toString(inputStream);
+			String response = new BufferedReader(
+				new InputStreamReader(inputStream, StandardCharsets.UTF_8)
+			).readLine();
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-engine-client/src/main/java/com/liferay/osb/faro/engine/client/web/client/ResponseErrorHandler.java:45]: @sillascavalcanti, what if the `BufferedReader` has more than one line?

**Review by marcosapmf — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-engine-client/src/main/java/com/liferay/osb/faro/engine/client/web/client/ResponseErrorHandler.java [hunk 2]
```diff
@@ -37,7 +38,16 @@ public void handleError(ClientHttpResponse clientHttpResponse)
 		}
 
 		try (InputStream inputStream = clientHttpResponse.getBody()) {
-			String response = IOUtils.toString(inputStream);
+			BufferedReader inputStreamRead = new BufferedReader(
+				new InputStreamReader(inputStream));
+			String input;
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-engine-client/src/main/java/com/liferay/osb/faro/engine/client/web/client/ResponseErrorHandler.java:43]: @sillascavalcanti can we inline this?

**Comment by marcosapmf** [comment]
resent to https://github.com/liferay-ac/liferay-portal/pull/1384

**Pull request closed without merging**